### PR TITLE
Enable CI checks also for main.py

### PR DESCRIPTION
**Pull request by Artanicus**
And fix a few things brought up by black and mypy.

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -57,25 +56,26 @@ def init(config: confuse.core.Configuration):
     logging.info(f"Starting Chiadog ({version()})")
 
     # Create log consumer based on provided configuration
-    chia_logs_config = config['chia_logs']
-    log_consumer = create_log_consumer_from_config(chia_logs_config)
+    chia_logs_config = config["chia_logs"]
+    log_consumer: LogConsumer = create_log_consumer_from_config(chia_logs_config)
     if log_consumer is None:
         exit(0)
```
Comment (martomi) [inline main.py:62]: nit: this check isn’t needed anymore.

Comment (Artanicus) [reply main.py:62]: Fixed.

**Comment by Artanicus** [comment]
Fixed the nit and threw in flake8 testing for main.py as well.

File: main.py (hunk 1 above)
Comment (martomi) [reply main.py:62]: ah, here I actually meant that we don’t need the if-statement since the function doesn’t anymore return `None`

Comment (martomi) [reply main.py:62]: But its a nit, happy to merge as-is

**Review by martomi — APPROVED** [review]

**Pull request merged**